presto-main/src/main/java/com/facebook/presto/sql/relational/Expressions.java

```diff
@@ -55,6 +59,17 @@ public static CallExpression call(String displayName, FunctionHandle functionHan
         return new CallExpression(displayName, functionHandle, returnType, arguments);
     }
 
+    public static CallExpression call(FunctionManager functionManager, String name, Type returnType, RowExpression... arguments)
+    {
+        return call(functionManager, name, returnType, ImmutableList.copyOf(arguments));
+    }
+
+    public static CallExpression call(FunctionManager functionManager, String name, Type returnType, List<RowExpression> arguments)
+    {
+        FunctionHandle functionHandle = functionManager.lookupFunction(name, fromTypes(arguments.stream().map(RowExpression::getType).collect(toImmutableList())));
+        return new CallExpression(name, functionHandle, returnType, arguments);
```

Comment:
- this should delegate to `call` two functions above.
- also, here `name` would be a bit confusing from `displayName`. Maybe @rongrong could provide suggestion.

Reply:
* will do
* Here, this utility is looking up a function by name and use the `name` as `displayName` since usually these two are the same. Probably `functionName` is better?